### PR TITLE
Registering team[JustCodeIt]

### PR DESCRIPTION
**Pull request by iamanantshukla**
# Application for Registering Team to CODE CAMP 1.0

This pull request template helps you complete an application to the **CODE CAMP 1.0 HACKATHON**. Use the checklist below to verify that you have followed the instructions correctly. 

**(Put x inside [] like [x])**

## Select the type of Pull Request

- [x] Registration Pull request.

- [] Submission Pull request.

## Checklist ✅  

- [x] I have read the instructions on the README file before submitting my application and gone through [CODE_OF_CONDUCT](). 
- [x] I have Submitted my team's details by making a folder of my team as instructed in [How to send Pull Request page](https://codecamphackathon.netlify.app/makingRegistration/).
- [x] I have used the Markdown file template to add my information for the Hackathon.
- [x] I have given all the necessary details as mentioned in template index.md file and the details are correct and best to my knowledge.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the Details added to the template will be used as a means of communication at the time of result declaration.
- [x] I have added the [event](https://calendar.google.com/calendar/r/eventedit/copy/MHUzM2pvaDQwM3R1cDdxMmdjMmQyYjZpaHAgN3NhMzI0N2FpMzkzczAzbWo0YWMxNWVzdG9AZw) to my calendar.

### CODE REVIEW

**Comment by Uyadav207** [comment]
@iamanantshukla , please fill the check list ASAP

File: content/teams/JustCodeIt/index.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+type: "team"                   
+title: JustCodeIt
+name: "JustCodeIt"
+github-link: "https://github.com/ "
+description: "hello I am Application Developer"
+Team-Quote: "just do it"
+Team_Tech_Stack: "Java, Android Studio, Firebase "
+Application_Type: "Application"
+No_Of_Member: ""
```
Comment (Uyadav207) [inline content/teams/JustCodeIt/index.md]: mention team strength members??

File: content/teams/JustCodeIt/index.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+_**Theme-Name**_ : Yoga Fitness App
+
+_**Application-Type**_ :   Mobile Application (The Application should be hosted and Bundled)
+
+_**Idea-Description**_ :   The restricted travel, panic over the risk of infection, continuous flow of negative news and scarcity of day-to-day material and resources due the corona pandemic are all adding to growing anxiety. Being confined to our homes can be mentally challenging. Besides staying connected to family and friends, and maintaining a healthy lifestyle, there is also one other thing that can help keep anxiety and mental health in check - yoga. So we are planning to make a app on yoga where user will be going through a series asanas and kriyas. It will show user how to do the asana properly and time it.
+
+_**Tech-Stack-Used**_ :   Java, Android Studio
+
+_**GitHub-Link**_ :   https://github.com/iamanantshukla/JustCodeIt/tree/master
```
Comment (Uyadav207) [inline content/teams/JustCodeIt/index.md]: remove tree/master

File: content/teams/JustCodeIt/index.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+_**Hosted-Link**_ :    Hosted link of project goes here. (Will submit in submission Pull Request)
+
+---
+
+# Member Details
+
+_**No of members**_ :
+
+3
+
```
Comment (Uyadav207) [inline content/teams/JustCodeIt/index.md]: mention correctly

File: content/teams/JustCodeIt/index.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+_**Name**_ : Devashish Tomar
+
+_**University-Name**_ : MNNIT Allahabad
+
+_**country**_ : India
+
+_**Technical-Skills**_ : Backend
+
+_**Specialization**_ :
```
Comment (Uyadav207) [inline content/teams/JustCodeIt/index.md]: please mention something

File: content/teams/JustCodeIt/index.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+_**Name**_ : Harshit Chauhan
+
+_**University-Name**_ : MNNIT Allahabad
+
+_**country**_ : India
+
+_**Technical-Skills**_ : Backend
+
+_**Specialization**_ :
```
Comment (Uyadav207) [inline content/teams/JustCodeIt/index.md]: ?? mention something

File: content/teams/JustCodeIt/index.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+_**University-Name**_ :
+
+_**country**_ :
+
+_**Technical-Skills**_ :
+
+_**Specialization**_ :
+
+_**GitHub-ID**_ :
```
Comment (Uyadav207) [inline content/teams/JustCodeIt/index.md]: Remove left over part

**Review by Uyadav207 — CHANGES_REQUESTED** [review]

**Comment by Uyadav207** [comment]
@iamanantshukla , please do the changes ASAP

**Review by Uyadav207 — APPROVED** [review]
@iamanantshukla ,
Hola  👋 ! Team  you have successfully completed the `First phase of the Hackathon`. ( We have successfully merged your PR for Registration ). 
Now we Request you to join your Team to our [**CODE CAMP GROUP 1**](https://chat.whatsapp.com/EowU6RVas3QCmtEs7q7L0H).
And you can start making your project from `21th July,2020`.
We have attached important Links. 

[**TimeLine**](https://codecamphackathon.netlify.app/TimeLine/)
[**Rule Book**](https://codecamphackathon.netlify.app/RuleBook/)
[**Prizes**](https://codecamphackathon.netlify.app/Prizes/)

See you there!
@ISTESRMNCR

**Comment by Uyadav207** [comment]
@iamanantshukla , please mention  correct theme, and join the team

**Comment by Uyadav207** [comment]
else not be able to merge your PR

**Comment by Uyadav207** [comment]
When done tag me for review pls ASAP

**Comment by Uyadav207** [comment]
@iamanantshukla , theme is wrong, do not put app name ,, put theme name like atma nirbhar bharat and from which our app belongs

**Review by Uyadav207 — APPROVED** [review]

**Comment by iamanantshukla** [comment]
> @iamanantshukla , theme is wrong, do not put app name ,, put theme name like atma nirbhar bharat and from which our app belongs

Sir is it ok now?

**Comment by Uyadav207** [comment]
@iamanantshukla , please join the group and text [`we are team team_name, we are in`]

Thanks and join team and follow instructions.

**Pull request merged**